### ISSUE TITLE
even if accounts disabled, gerbera shows the login mask, user locked out.

### ISSUE THREAD

**Issue by vasyugan** [gerbera/gerbera#159]
The configfile says:
```

  <ui enabled="yes" show-tooltips="yes">
      <accounts enabled="no" session-timeout="30">
        <account user="gerbera" password="gerbera"/>
      </accounts>
    </ui>

```

Yet I get the login mask, also, with the given username and password I cannot login, submit does absolutely nothing. Changing accounts to enabled changes nothing about this. I am on arch linux arm, raspberry pi 3 gerbera 1.0.0-3  from AUR

I find absolutely nothing related in the system log.

**Comment by elmodaddyb** [comment]
@vasyugan - We had an issue a while back on this regarding User Login on the web application for ARM based compilations. --> #105 

Can you try building from **latest master**?  To see if ARM issues are resolved?

**Comment by vasyugan** [comment]
This seems to be fixed in Master, thanks!

**Comment by v00d00** [comment]
Closing this one too, fixed in 1.1/master.
